packages/shared-ux/chrome/navigation/src/ui/v2/default_navigation.test.helpers.ts

```diff
@@ -0,0 +1,449 @@
+/*
```

Comment:
Will this be placed in a "mocks" directory in the next PR when the `v2` prefix goes away?

Reply:
That's not what I had in mind. Those are actually helpers (constants in this case) for the tests. They are not mocks. 
Instead of bloating the test file with a huge object (or sometimes utility functions) I put those in a file next to the test with suffix `helper`.